transport_udp_test.go

```diff
@@ -59,6 +59,7 @@ func TestEmitBatchOverhead(t *testing.T) {
 	client := j.NewAgentClientFactory(transport, protocolFactory)
 
 	span := &Span{operationName: "test-span", tracer: jaegerTracer}
+	span.context.samplingState = &samplingState{}
```

Comment:
this is kind of unfortunate that we have to do it, but I don't see a much cleaner way (other than having proper Span constructor)

Reply:
at least in this test file you could DRY it by using a helper function instead of two lines everywhere